### DISCUSSION TITLE
Vertical tabs

### DISCUSSION THREAD

**Discussion by georgedobre** [tabler/tabler#1900] [Ideas]
Hi guys,

I've been trying to find an example of some vertical tabs to no avail, so if there is one apologies.
I went ahead and done my own with the css provided, if someone is interested i have uploaded a gist at https://gist.github.com/georgedobre/73bdea0f8ff53e1a784f641fd86475ec

https://github.com/tabler/tabler/assets/10349279/86fc2c88-4092-46dd-8485-c2e319d0fa52

**Comment by iamkwasi** [comment]
Thanks really. I tried using a jquery vertical tab, but anytime I tried, the tabs become horizontal for some reason. This is really helpful.

**Reply by georgedobre** [reply]
glad you found it useful 👍
